src/main/java/digital/slovensko/autogram/ui/gui/GUI.java

```diff
@@ -150,7 +150,7 @@ public void pickTokenDriverAndThen(List<TokenDriver> drivers, Consumer<TokenDriv
     @Override
     public void requestPasswordAndThen(TokenDriver driver, Consumer<char[]> callback) {
         if (!driver.needsPassword()) {
-            callback.accept(null);
+            callback.accept("".toCharArray());
```

Comment:
?

Reply:
null hodí error, ak sa niekam pošle. Táto vetva sa zatiaľ veľmi nepoužívala, lebo pri FakeTokene sa v tej metóde aj tak ignoroval parameter. A iný driver sme bez hesla nemali. Toto funguje všeobecnejšie - nerozbije to null-om nič.